ekncontent/ekncontent/eknc-utils.h

```diff
@@ -8,6 +8,19 @@
 
 G_BEGIN_DECLS
 
+/**
+ * EkncContentObjectError:
+ * @EKNC_CONTENT_OBJECT_ERROR_BAD_FORMAT: unexpected format in metadata json
+ *
+ * Error enumeration for object model creation.
+ */
+typedef enum _EkncContentObjectError {
+  EKNC_CONTENT_OBJECT_ERROR_BAD_FORMAT,
+} EkncContentObjectError;
+
+#define EKNC_CONTENT_OBJECT_ERROR eknc_content_object_error_quark ()
+GQuark eknc_content_object_error_quark (void);
```

Comment:
Should this be moved up into an earlier commit so that you can use the error quark there as well? There are some more instances where you just passed 0